docs/index.rst

```diff
@@ -5,75 +5,71 @@ Introduction
+maintained by the `Stellar Development Foundation`_ (SDF). Using Polaris, you can run a web
+server supporting any combination of SEP-1, 6, 10, 12, and 24.
 
-Polaris is not a library or a framework; its an `extendable`
-`django-app`_.  Like many django apps, it comes with fully-implemented
-endpoints, templates, and database models.
+While Polaris handles the majority of the business logic described in each SEP, there are
+pieces of functionality that can only be implemented by the developer using Polaris.
+For example, only an anchor can implement the integration with their partner bank.
 
-To use Polaris, developers must implement it's provided
-:doc:`integrations points </integrations/index>`. These integration points
-allow developers to inject their own business logic into the transaction
-processing flow, customize their stellar.toml, and more.
+This is why each SEP implemented by Polaris comes with a programmable interface, or
+integrations, for developers to inject their own business logic.
 
-The SDF maintains a Polaris server at that can be tested using the SEP-24
-`demo client`_ and is completely open source and available on the SDF's github_.
+Polaris is completely open source and available on github_. The SDF also runs a reference
+server using Polaris that can be tested using our `demo client`_.
+
+The instructions below outline the common set up needed for any Polaris deployment, but
+each SEP implementation has its own configuration and integration requirements. These
+requirements are described in the documentation for each SEP.
 
 Installation and Configuration
 ==============================
 
-.. _CLI tool: https://github.com/msfeldstein/create-stellar-token
-.. _Static Files: https://docs.djangoproject.com/en/2.2/howto/static-files/
+.. _Django docs: https://docs.djangoproject.com/en/3.0/
+
+These instructions assume you have already set up a django project. If you haven't,
+take a look at the `Django docs`_.
 
 First make sure you have ``cd``'ed into your django project's main directory
 and then run
 ::
 
     pip install django-polaris
 
-Configuring settings.py
-^^^^^^^^^^^^^^^^^^^^^^^
+Settings
+^^^^^^^^
 
-Add the following to ``INSTALLED_APPS`` in settings.py. Any app that overrides
-a static asset in Polaris should be listed `before` "polaris". This ensures that
-django will find your asset before the Polaris default.
+Add the following to ``INSTALLED_APPS`` in settings.py.
 ::
 
     INSTALLED_APPS = [
         ...,
-        "django.contrib.staticfiles",
         "corsheaders",
         "rest_framework",
-        "sass_processor",
         "polaris",
     ]
 
-Add Polaris' :doc:`PolarisSameSiteMiddleware </middleware/index>`,
-and ``CorsMiddleware`` to your ``settings.MIDDLEWARE``.
-``SessionMiddleware`` must be listed `below` ``PolarisSameSiteMiddleware``.
+Add ``CorsMiddleware`` to your ``settings.MIDDLEWARE``. It should be listed above
```

Comment:
Yes, its possible using a django signal the package provides: https://github.com/adamchainz/django-cors-headers#signals

However I need to test how this works. If we add a signal to allow all origins for Polaris URLs, the user would have to explicitly list origins that are acceptable to all other endpoints. What if the user has another set of URLs that should allow all origins? 

If it is possible to register multiple signals so users could define additional filters then this works. But if only one signal function can be registered to accept or deny origins, it probably makes sense for the user to write it themselves. Otherwise we'd be requiring them to explicitly whitelist origins or find a workaround.

Reply:
Either way, the user will still need to add `CorsMiddleware` so Polaris can allow all origins.

Let me experiment with different configurations. For now, since this is already how it works, I don't think it should block this PR.

Reply:
ok this isn't relevent to this PR no need to make it bigger, but something to consider for long term maintenance

Reply:
#195